sdk/scvmm/arm-scvmm/_meta.json

```diff
@@ -0,0 +1,8 @@
+{
+  "commit": "5d1d77e56c33e7d2522f489b3ab2b572c3d6a49a",
+  "readme": "specification/scvmm/resource-manager/readme.md",
+  "autorest_command": "autorest --version=3.7.3 --typescript --modelerfour.lenient-model-deduplication --head-as-boolean=true --license-header=MICROSOFT_MIT_NO_VERSION --generate-test --typescript-sdks-folder=D:\\work\\azure-sdk-for-js ..\\azure-rest-api-specs\\specification\\scvmm\\resource-manager\\readme.md --use=@autorest/typescript@6.0.0-alpha.19.20220408.1 --generate-sample=true",
+  "repository_url": "https://github.com/Azure/azure-rest-api-specs.git",
+  "release_tool": "@azure-tools/js-sdk-release-tools@2.2.6",
+  "use": "@autorest/typescript@6.0.0-alpha.19.20220408.1"
```

Comment:
Could you update your lib of `@autorest/typescript` to latest alpha version? We migrated the test recorder to latest one.